### PR TITLE
Main screen

### PR DESCRIPTION
**Pull request by agrawalshanu708**
create Main screen with Greeting message
Features Added
weather 
user can add his task
get a new background image after every refresh

### CODE REVIEW

File: src/Components/Clock/Clock.jsx [hunk 1]
```diff
@@ -1,8 +1,105 @@
...
+
+const timeFunction = () =>{
+    let date = new Date();
+        let hours = date.getHours();
+        let minutes = ('0'+date.getMinutes()).slice(-2);
+        const currentTime = `${(hours % 12) || 12}:${minutes}` ;
+        setTime(currentTime);
+}
+      
+      const greetingMessage =() =>{
```
Comment (harshitpaliwal95) [inline src/Components/Clock/Clock.jsx:32]: good work

File: src/Components/Weather/Weather.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    font-weight: 500;
+}
+.widget-status {
+    font-size: 2rem;
+    /* padding-top: 5px; */
+}
+
+
+
+
```
Comment (harshitpaliwal95) [inline src/Components/Weather/Weather.css:18]: remove commented code

File: src/Components/Weather/Weather.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    margin-right:5rem
+}
+
+.widget-location {
+    font-size: 2.5rem;
+    font-weight: 500;
+}
+.widget-status {
+    font-size: 2rem;
+    /* padding-top: 5px; */
```
Comment (tushargoyal812) [inline src/Components/Weather/Weather.css:13]: remove comments if not needed

File: src/Components/Weather/Weather.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useEffect, useState } from 'react'
+import axios from 'axios';
+import "./Weather.css"
+// import {loaderGif} from '../../assets';
```
Comment (tushargoyal812) [inline src/Components/Weather/Weather.jsx:5]: remove comments

**Review by harshitpaliwal95 — APPROVED** [review]
great work Shanu easy to read 
good file folder structure

File: src/hooks/useBackgroundImage.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    const ACCESS_KEY = '27443352-bb87137647822329acb8af729';
+    const endpointLocation = `https://pixabay.com/api/?key=${ACCESS_KEY}&q=landscape&image_type=photo&orientation=horizontal&category=${randomBgCategory}&per_page=50&order=popular`;
+
+
+    const fetchBgImage = async() => {
+        try {
+            const res = await get(endpointLocation);
+            setBgURL(res.data.hits[Math.floor(Math.random() * res.data.hits.length)].largeImageURL)
+        } catch(error) {
+            console.log(error.message);
```
Comment (tushargoyal812) [inline src/hooks/useBackgroundImage.jsx:22]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
code is clean and to the point
folder structure is good.
variable naming is good.

**Pull request merged**